app/jobs/populate_register_data_in_db_job.rb

```diff
@@ -1,36 +1,13 @@
 class PopulateRegisterDataInDbJob < ApplicationJob
   queue_as :default
 
-  module Exceptions
-    class FrontendInvalidRegisterError < StandardError; end
-  end
-
   def perform(register)
-    Delayed::Worker.logger.info("Updating #{register.name} in database")
-    begin
-    # Initialize client and download / update data
-      registers_client_manager = RegistersClient::RegisterClientManager.new({
-        api_key: Rails.configuration.try(:registers_api_key)
-      }.compact)
-      register_client = registers_client_manager.get_register(register.name.parameterize, register.register_phase.downcase, data_store: PostgresDataStore.new(register))
-      register_url = register_client.instance_variable_get(:@register_url)
-      register_client.refresh_data
-    rescue InvalidRegisterError => e
-    # If register data is invalid we want to delete existing entries and records to force a full reload
-      ActiveRecord::Base.transaction do
-        Record.where(register_id: register.id).delete_all
-        Entry.where(register_id: register.id).delete_all
-        register.root_hash = nil
-        register.save
-      end
-      raise Exceptions::FrontendInvalidRegisterError, "#{register.name}: #{e}"
-    end
+    logger.info("Updating #{register.name} in database")
```

Comment:
can we check this still logs correctly when performed on the queue service?

Reply:
Have replaced this with DelayedJob.logger